game/src/main/scala/io/aigar/game/EntityContainer.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace.Vector2
+
+
+trait EntityContainer {
+  def respawn(grid: Grid, players: List[Player], respawnRetryAttempts: Int): Option[Vector2] = {
+    var newPosition = grid.randomPosition
```

Comment:
If we're doing it this way then it's all good! I misunderstood the original goal, my bad.